app/templates/views/service-join-request-rejected.html

```diff
@@ -0,0 +1,18 @@
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+
+{% block service_page_title %}
+  You cannot let {{requested_by.name}} join your service
+{% endblock %}
+
+{% block maincolumn_content %}
+
+  {{ page_header('You cannot let {} join your service'.format(requested_by.name)) }}
+
+  <p class="govuk-body">
+    {{ rejected_by.name }} already refused their request on {{ rejected_at.strftime('%-d %B') }}.
```

Comment:
Use a formatter rather than hard-code string formatting:
```suggestion
    {{ rejected_by.name }} already refused their request on {{ rejected_at|format_date_short }}.
```

Comes from:
https://github.com/alphagov/notifications-admin/blob/a5be2df07d3b97dc981855fd5d048b56456a7c3b/app/formatters.py#L97-L98

This is:
- easier to read
- handles timezones properly